compose/tests/galaxy-workflow-test/Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM alpine:3.11
 
-ENV TEST_REPO=${TEST_REPO:-https://github.com/usegalaxy-eu/workflow-testing} \
-    TEST_RELEASE=${TEST_RELEASE:-master}
+ENV TEST_REPO=${TEST_REPO:-https://github.com/jonas27/workflow-testing} \
```

Comment:
BEFORE MERGE: Check if this is ok!
- It comments out the data types in the workflows.
-   If not the tests fail on the metadata and throw an error on samtools wanting to scan an empty file.